### PR TITLE
Greg/qajan16

### PR DESCRIPTION
**Pull request by GodGreg**

### CODE REVIEW

File: src/Components/FormCreation/CreateEditForm.js [hunk 1]
```diff
@@ -662,40 +663,20 @@ function CreateEditForm({ programId }) {
...
-                        handleDescriptionUpdate={handleDescriptionUpdate}
-                        handleSectionTypeUpdate={handleSectionTypeUpdate}
-                        handleMoveSection={handleMoveSection}
-                        handleDeleteSection={handleDeleteSection}
-                        setShowMoveSectionsDialog={setShowMoveSectionsDialog}
-                      />
-                    </FormWrapper>
-                  </div>
-                ))}
-            </DragDropContext>
```
Comment (ahmedhamodi) [inline src/Components/FormCreation/CreateEditForm.js:688]: On first code pass, seems a bit weird this is being deleted? Will see what it looks like in app after review.

**Review by ahmedhamodi — COMMENTED** [review]
First code pass. Going to test in app.

File: src/Components/FormCreation/CreateEditForm.js (hunk 1 above)
Comment (GodGreg) [reply src/Components/FormCreation/CreateEditForm.js:688]: It was copy and pasted by Guy for the delete section functionality. When the section was deleted, but still able to be undone this code executed and showed double the form.

**Review by ahmedhamodi — APPROVED** [review]
Went through tickets in review of the bug bash and it looks to all be implemented.

**Pull request merged**